client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-sites-dataviews.tsx

```diff
@@ -42,22 +43,28 @@ export const JetpackSitesDataViews = ( {
 	const sitesPerPage = dataViewsState.perPage > 0 ? dataViewsState.perPage : 20;
 	const totalPages = Math.ceil( totalSites / sitesPerPage );
 
-	const sites = useFormattedSites( data?.sites ?? [] ).reduce< SiteData[] >( ( acc, item ) => {
-		item.ref = item.site.value.blog_id;
-		acc.push( item );
-		// If this site has an error, we duplicate this row - while changing the duplicate's type to 'error' - to display an error message below it.
-		if ( item.site.error ) {
-			acc.push( {
-				...item,
-				site: {
-					...item.site,
-					type: 'error',
-				},
-				ref: `error-${ item.ref }`,
-			} );
-		}
-		return acc;
-	}, [] );
+	const possibleSites = data?.sites ?? [];
+	const connectionStatus = useFetchTestConnections( true, possibleSites );
```

Comment:
First parameter here is `isPartnerOAuthTokenLoaded` which is guarding the request. I know we assume it false in other places within A4A but without it this endpoint won't work.